build.py

```diff
@@ -380,16 +380,15 @@ def build(version=None,
         if len(tags) > 0:
             build_command += "-tags {} ".format(','.join(tags))
 
+        ldflags = "-ldflags=\"-X {package}.Version={version} -X {package}.GitBranch={branch} -X {package}.GitCommit={commit}\" "

```

Comment:
The public part is put forward, which is more concise, but logically not as clear as the previous if ... else... structure. My suggestion is to maintain the original structure, what do you think?

Reply:
Actually, this is the `else` part, I set it to default to avoid using an `else'.

This style is proposed by the [uber coding guide line](https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else), which has been widely adopted (although it is for Go, I think it can be applied to Python), and personally, I prefer this style too.

But I think both are acceptable and will make the change, along with another minor improvement: `-a -installfuffix cgo` is not part of `ldflags`.

Reply:
it's okay, shall I merge it?